.github/workflows/test.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node: [18, 20, 22]
+        node: ['18.16.0', 20, 22]
```

Comment:
Let's update the engines field in package.json to match.

Reply:
Oh or maybe with this change we actually end up supporting back more so we can leave engines where it is at now.